### DISCUSSION TITLE
Problema installazione Sonoff Zigbee 3.0 USB Dongle Plus

### DISCUSSION THREAD

**Discussion by fonzarello74** [indomus/forum#3321] [HUB: Home Assistant]
Ciao a tutti,
premesso che sono alle prime armi con HA, ho configurato un Raspberry 4 con Raspberry Pi OS e HA Core su docker, tutto funziona regolarmente, non riesco però a configurare il dongle USB di Sonoff per Zigbee. Ho seguito questa guida: https://indomus.it/guide/come-installare-e-configurare-silicon-labs-multiprotocol-per-gestire-coordinator-multipan-rcp/

ma quando vado a lanciare il comando:
```
docker run -d --privileged --network host -v ./data/:/data --name multipan --device /dev/ttyACM0 --cap-add SYS_ADMIN --cap-add NET_ADMIN -e DEVICE=/dev/ttyACM0 -e BAUDRATE=460800 -e CPCD_TRACE=false -e FLOW_CONTROL=false -e NETWORK_DEVICES=0 -e OTBR_ENABLE=1 -e BACKBONE_IF=wlp2s0 -e OTBR_LOG_LEVEL=debug -e OTB_FIREWALL=1 -e OTBR_REST_LISTEN_PORT=8081 -e OTBR_WEB_PORT=8086 -e NETWORK_DEVICE= -e EZSP_LISTEN_PORT=9999 -e AUTOFLASH_FIRMWARE=0 -e FIRMWARE= ghcr.io/b2un0/silabs-multipan-docker:latest
``` 

oppurtanamente modificato con il mio "device" e "backbone_if" mi restituisce questo errore:
"Unable to find image 'ghcr.io/b2un0/silabs-multipan-docker:latest' locally
latest: Pulling from b2un0/silabs-multipan-docker
docker: manifest unknown."

 Grazie

**Comment by KC-inDomus** [comment]
il problema è che per la tua versione di sistema operativo/architettura, online non c'è l'immagine docker che ti serve.
chiedi [qui](https://github.com/b2un0/silabs-multipan-docker/discussions)

**Reply by fonzarello74** [reply]
Grazie per la risposta,
secondo te su raspberry 4 quale sarebbe la migliore installazione da fare? Considerando che ho appena installato non mi preoccuperebbe ricominciare da capo.
Pensi che usando il suo sistema operativo ottengo una piattaforma più plug & play?
Grazie ancora

**Reply by KC-inDomus** [reply]
beh solitamente la armv8 64 bit, però verifica sul manifest quali sono le effettive versioni disponibili dell'immagine docker...

**Reply by fonzarello74** [reply]
Grazie mille, faccio una verifica.
